dataland-frontend/tests/e2e/fixtures/CsvUtils.ts

```diff
@@ -1,30 +1,79 @@
-import { EuTaxonomyDataForFinancialsFinancialServicesTypesEnum } from "@clients/backend";
-
-export function getAttestation(attestation: string) {
-  if (attestation === "LimitedAssurance") {
-    return "limited";
-  } else if (attestation === "ReasonableAssurance") {
-    return "reasonable";
-  } else {
-    return "none";
+import {
+  AssuranceDataAssuranceEnum,
+  EuTaxonomyDataForFinancialsFinancialServicesTypesEnum,
+  YesNo,
+  YesNoNa,
+} from "@clients/backend";
+import Big from "big.js";
+import { humanizeString } from "../../../src/utils/StringHumanizer";
+
+export function getAssurance(assurance: AssuranceDataAssuranceEnum | undefined): string | undefined {
+  switch (assurance) {
+    case undefined:
+    case null:
+      return undefined;
+    case AssuranceDataAssuranceEnum.LimitedAssurance:
+      return "limited";
+    case AssuranceDataAssuranceEnum.ReasonableAssurance:
+      return "reasonable";
+    case AssuranceDataAssuranceEnum.None:
+      return "none";
+  }
+  throw Error(`Unknown assurance type ${assurance}`);
+}
+
+export function getFiscalYearDeviation(isdeviation: YesNo | undefined): string | undefined {
+  switch (isdeviation) {
+    case undefined:
+    case null:
+      return undefined;
+    case YesNoNa.Yes:
+      return "Deviation";
+    case YesNoNa.No:
+      return "No Deviation";
   }
+  throw Error(`Unknown yesno type ${isdeviation}`);
 }
 
-export function getCompanyType(type: EuTaxonomyDataForFinancialsFinancialServicesTypesEnum): string {
+export function getCompanyTypeHeader(type: EuTaxonomyDataForFinancialsFinancialServicesTypesEnum): string {
   switch (type) {
     case EuTaxonomyDataForFinancialsFinancialServicesTypesEnum.AssetManagement:
       return "Asset Management Company";
     case EuTaxonomyDataForFinancialsFinancialServicesTypesEnum.CreditInstitution:
       return "Credit Institution";
     case EuTaxonomyDataForFinancialsFinancialServicesTypesEnum.InsuranceOrReinsurance:
       return "Insurance/Reinsurance";
+    case EuTaxonomyDataForFinancialsFinancialServicesTypesEnum.InvestmentFirm:
+      return "Investment Firm";
   }
   throw Error(`Unknown FS type ${type}`);
 }
 
-export function decimalSeparatorConverter(value: number | undefined): string {
-  return value === undefined ? "" : value.toString().replace(".", ",");
+export function humanizeOrUndefined(stringToHumanise: string | undefined): string | undefined {
+  if (stringToHumanise == undefined) return undefined;
+  return humanizeString(stringToHumanise!!);
 }
+
+export function getCompanyTypeCsvValue(type: EuTaxonomyDataForFinancialsFinancialServicesTypesEnum): number {
```

Comment:
The other way is implemented in the csv parser; one might look into combining these (and also all further mappings in the StringHumanizer)

Reply:
Not worth the effort dealing with the different file types right now